types/graphql-schema-linter/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for graphql-schema-linter 0.2
+// Project: https://github.com/cjoudrey/graphql-schema-linter
+// Definitions by: Gago <https://github.com/gagoar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Configuration } from './lib/configuration';
+import { validateSchemaDefinition } from './lib/validator';
+
+export interface configuration {
```

Comment:
This needs to be value and not interface based on https://github.com/cjoudrey/graphql-schema-linter/blob/402b0a6dd610b00ba346c422e8558dd4dc15b634/src/index.js#L3